.github/workflows/ci-config.yml

```diff
@@ -0,0 +1,45 @@
+name: CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+  schedule:
+    - cron: '0 0 * * *'
+
+jobs:
+  run_tests:
+    runs-on: ubuntu-latest
+    services:
+      redis:
+        image: redislabs/redisai:edge-cpu-bionic
```

Comment:
Is this a suggestion? I chose edge because I thought that we want to know if master is breaking the demo. In the docs, we should tell users to use latest...